ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vlan-interfaces.md

```diff
@@ -164,6 +166,12 @@ interface Vlan24
    ipv6 virtual-router address 1b11:3a00:22b0:6::1
    ip address virtual 10.10.24.1/24
 !
+interface Vlan25
+   description SVI Description
+   no shutdown
+   ipv6 virtual-router address 1b11:3a00:22b0:16::15/64
```

Comment:
missing update to the interface table. You should add the addresses from the new model to the existing table field. Check the v4 table for inspiration :)